browser/module.go

```diff
@@ -41,16 +46,38 @@ func New() *RootModule {
 
 // NewModuleInstance implements the k6modules.Module interface to return
 // a new instance for each VU.
-func (*RootModule) NewModuleInstance(vu k6modules.VU) k6modules.Instance {
+func (m *RootModule) NewModuleInstance(vu k6modules.VU) k6modules.Instance {
 	return &ModuleInstance{
 		mod: &JSModule{
-			Chromium: mapBrowserToGoja(moduleVU{vu}),
-			Devices:  common.GetDevices(),
-			Version:  version,
+			Chromium: mapBrowserToGoja(moduleVU{
+				VU:   vu,
+				root: m,
+			}),
+			Devices: common.GetDevices(),
+			Version: version,
 		},
 	}
 }
 
+// pids returns the launched browser process IDs.
+func (m *RootModule) pids() []int {
+	m.muBrowserProcessIDs.RLock()
+	defer m.muBrowserProcessIDs.RUnlock()
+
+	pids := make([]int, len(m.browserProcessIDs))
+	copy(pids, m.browserProcessIDs)
+
+	return pids
+}
+
+// addPid keeps track of the launched browser process IDs.
+func (m *RootModule) addPid(pid int) {
```

Comment:
Should this be `addPID` ?

Reply:
Pid is a valid convention in Go. See [this](https://pkg.go.dev/os#Process) for an example.

```go
type Process struct {
	Pid [int](https://pkg.go.dev/builtin#int)
	// contains filtered or unexported fields
}
```